### DISCUSSION TITLE
Make `> Export list` export absolute paths

### DISCUSSION THREAD

**Discussion by cgebbe** [jarun/nnn#1683] [Q&A] [answered]
Currently, `> Export list` exports the list of filenames / dirnames. Is it possible to export absolute filepaths / dirpaths instead?

I know there's already the "copy-selection-to-clipboard" feature, but it requires installing the plugins and an X11-server when working on machines connected to SSH.

**Comment by lawnowner** [comment, marked as answer]
`ar` instead of `>` lists absolute paths since selected files can be from different directories.

**Comment by jarun** [comment]
Select and try edit selection with <kbd>E</kbd>.

**Reply by jarun** [reply]
Just check it out with tmp files first. Not sure what's the outcome of trying to save the file in some other name.
